Identification-and-Assessment.tex

```diff
@@ -2,26 +2,26 @@ \section{Risk \& Opportunity Identification and Assessment}
 \label{sec:id-assess}
 
 This section describes who and how risks and opportunities are identified and assessed.
-This is currently written when using the NOIRLab Risk Tool.
+This is currently written assuming that the Risk Register is implemented in NOIRLab's ATS Risk Tool.
 
 \begin{enumerate}
-	\item The managing group creates a risk or opportunity in the ATS Risk Tool as a "Candidate."
+	\item The managing group creates a risk or opportunity in the Risk Register as a "Candidate."
 	\item RROB reviews content, then either asks for more information or sets risk to "Active."
-	\item After 
+	\item After
 \end{enumerate}
 
 \subsection{Create Risk Item}
 
-Minimum content before saving in Risk Tool:
+Minimum content before saving in the Risk Register:
 \begin{itemize}
 	\item Project --- Rubin Operations
 	\item Risk Type --- Threat or Opportunity
```

Comment:
Note that this line, which we inherit from our Risk Register, is inconsistent with the Definitions section, which wants to equate "Risk" with "Threat". For as long as our Risk Register is managed by NOIRLab using the current tool, I think we have to use that tool's terminology, and adopt the position that "risks" can have either negative impacts (in which case they are "threats") or positive impact (in which case they are "opportunities"). This logic leads to a number of other consequences: for example, the title of this document should be the "Risk Management Plan" not the "Risk and Opportunity Management Plan." (I suppose it could be "Threat and Opportunity Management Plan".) Similarly, the approving body should be the "Risk Board" not the "Risk and Opportunity Board." Leaving this here for @leannep to consider - it's a little jarring having the Risk Management Plan try and use different terminology to the Risk Register.

Reply:
As this cannot be addressed during PR#5, please make a github issue to capture the needed change.